app/supabaseClient.ts

```diff
@@ -0,0 +1,6 @@
+import { createClient } from '@supabase/supabase-js'
+
+const supabaseUrl = import.meta.env.VITE_SUPABASE_URL ?? ''
+const supabaseAnonKey = import.meta.env.VITE_SUPABASE_ANON_KEY ?? ''
```

Comment:
**issue**: Ensure that environment variables are correctly set and handled to avoid runtime errors.